GoToProject/src/library/methods/firebaseUploadImage..js

```diff
@@ -0,0 +1,17 @@
+import storage from '@react-native-firebase/storage';
+
+export const uploadImage = async (uri, filename) => {
+  const reference = storage().ref(filename);
+  const task = reference.putFile(uri);
+  
+  task.then(() => {
+    console.log('Image uploaded to the bucket!');
```

Comment:
Remember to avoid leaving console.log() in your code when uploading to GitHub. If you need to display any messages, especially for the user, you can do so using the [Alert.alert()](https://reactnative.dev/docs/alert) element of React Native.

Reply:
thank you for the comment